src/Symfony/Component/ErrorHandler/Resources/views/trace.html.php

```diff
@@ -25,6 +25,9 @@
                 <span class="trace-method"><?= $trace['function']; ?></span>
             <?php } ?>
             (line <?= $lineNumber; ?>)
+            <span class="icon icon-copy hidden" data-clipboard-text="<?php echo implode(\DIRECTORY_SEPARATOR, $filePathParts).':'.$lineNumber; ?>">
```

Comment:
I think the `data-clipboard-text` should contain the full path to the file - *not* the formatted path that links to the IDE.
This means this should contain `$trace['file']`, html-escaped.

Reply:
I'm not sure about that... when running app in docker, absolute path is useless, while relative pat works in IDE, and console (when CWD=root of project)

Reply:
Ah, you're right. We do have a way to map paths from docker to host, but it is bound to the IDE URLs. Can't we use that map somehow to compute absolute paths for the host?